WordPress/UITestsFoundation/TestObserver.swift

```diff
@@ -0,0 +1,63 @@
+import XCTest
+
+class TestObserver: NSObject, XCTestObservation {
+    override init() {
+            super.init()
+            XCTestObservationCenter.shared.addTestObserver(self)
+        }
+
+    func testBundleWillStart(_ testBundle: Bundle) {
+        // Code added here will run only once before all the tests
+        executeWithRetries(disableAutoFillPasswords)
+    }
+
+    func executeWithRetries(_ operation: () -> Bool) {
+        var retryCount = 3
+
+        while !operation() && retryCount > 0 {
+            retryCount -= 1
+        }
+    }
```

Comment:
Retrying to be on the safer side as all the tests would fail in case this is not disabled. I have seen it happening a couple times in a week. 
I tried to make it reusable as we might need to change other setting in the future.